examples/data/FASTAnalyzer_results_A.csv

```diff
@@ -0,0 +1,19 @@
+Class,Goal,Analysis variable,heatPumpSystem.declination,vol.V,heaCap.C,rad.n,theCon.G
+global,all,S1,3.699703925116529e-05,0.001281425172057828,0.0015227349583394145,0.0008281593221427361,0.8366451874300566
+global,all,ST,0.10796269929100588,0.12727189909324055,0.1155495847504745,0.14553950605124466,0.9611281699385661
+global,Electricity,S1,0.00031747319966370904,0.00024353085199873387,0.0003764898090155623,0.000135902101349077,0.2273758074248678
+global,Electricity,ST,0.7453376933545235,0.743962586600946,0.7341629863644938,0.6621874987030962,0.9102144331281674
```

Comment:
Are all these results automatically generated or a required input for the examples? If it's an input, what do you need it for aside from plotting?

Reply:
These results were created separately with a high sample number to showcase the right results in the example. The creation in the example would take too long, so there are only very small sample numbers used.